test/functional/core/job_spec.lua

```diff
@@ -101,13 +101,15 @@ describe('jobs', function()
   end)
 
   it('returns -1 when target is not executable #5465', function()
-    if helpers.pending_win32(pending) then return end
     local function new_job()
       return eval([[jobstart('')]])
     end
     local executable_jobid = new_job()
-    local nonexecutable_jobid = eval(
-      "jobstart(['./test/functional/fixtures/non_executable.txt'])")
+    local nonexecutable = './test/functional/fixtures'
+    if not iswin() then
+      nonexecutable = nonexecutable .. '/non_executable.txt'
+    end
+    local nonexecutable_jobid = eval("jobstart(['"..nonexecutable.."'])")
```

Comment:
ternary is less noisy, here.

```lua
local nonexecutable_jobid = (iswin() 
  and eval("jobstart(['./test/functional/fixtures'])")
  or eval("jobstart(['./test/functional/fixtures/non_executable.txt'])"))
```

Reply:
Will that run 2 jobs?

Reply:
No, if `iswin()` is false then the `and ...` is skipped. And `eval()` cannot return `nil`, unless something else is broken.

Reply:
What if `jobstart` returns 0?